Src/DependencyCollector/Shared/DependencyTrackingTelemetryModule.cs

```diff
@@ -177,9 +183,12 @@ protected virtual void Dispose(bool disposing)
         private void InitializeForFrameworkEventSource()
```

Comment:
The FrameworkHttpEventListener path is unchanged. FrameworkHttpProcessing.OnBeginHttpCallback and OnEndHttpCallback are still there. The worst case is, you just get what you get today.